specification/VRMC_springBone-1.0_draft/schema/VRMC_springBone.joint.schema.json

```
@@ -28,7 +36,7 @@
     },
     "dragForce": {
       "type": "number",
-      "description": "Air resistance. Deceleration force"
+      "description": "Air resistance. Deceleration force."
     }
   },
   "required": [
```

[MUST]

```suggestion
  "required": [
    "node",
```

および、 `stiffness`, `gravityPower`, `dragForce` が必須となるなら、 `hitRadius` も必須で良い気がしています。